cli/testdata/coder_delete_--help.golden

```diff
@@ -8,6 +8,11 @@ USAGE:
   Aliases: rm
 
 OPTIONS:
+      --debug-provisioner bool
```

Comment:
Should we keep this flag hidden? I believe this is only for template admins as it may leak some secrets in the TF verbose mode.

Reply:
I'm happy to hide it :+1: